docs/source/tools/visual.rst

```diff
@@ -0,0 +1,32 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+DAML Contract Visulization
+#########
```

Comment:
This isn't gonna work, unfortunately - needs to be as long as the line above

Reply:
😢